website/views/options/social/index.jade

```diff
@@ -113,3 +113,17 @@ script(type='text/ng-template', id='partials/options.social.html')
   .tab-content
     .tab-pane.active
       div(ui-view)
+
+script(type='text/ng-template', id='modals/copyChatToDo.html')
```

Comment:
@crookedneighbor - So do I :/  What is being linked in the notes that
requires clicking? Is it only if the zapped chat message contained a link,
or does it link back to the original message? I'm not fully familiar with
how this feature is currently implemented, so I can't say with total
certainly that this is an essential feature. I'd love to see some more
screenshots if possible.

On Mon, Apr 27, 2015 at 11:01 AM, Blade Barringer <notifications@github.com>
wrote:

> In website/views/options/social/index.jade
> <https://github.com/HabitRPG/habitrpg/pull/5087#discussion_r29172639>:
>
> > @@ -113,3 +113,17 @@ script(type='text/ng-template', id='partials/options.social.html')
> >    .tab-content
> >      .tab-pane.active
> >        div(ui-view)
> > +
> > +script(type='text/ng-template', id='modals/copyChatToDo.html')
>
> @lemoness <https://github.com/lemoness> Thoughts on showing the notes
> popover on click rather than hover?
>
> I'm anticipating numerous bug reports from people saying their notes won't
> appear if we make that switch.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/5087/files#r29172639>.
>